doggos/fuzzy_sets/degree/membership_degree.py

```diff
@@ -0,0 +1,5 @@
+from abc import ABC
+
+
+class MembershipDegree(ABC):
```

Comment:
Before class declaration you should use @dataclass decorator. You can read about it on https://docs.python.org/3/library/dataclasses.html